scripts/test/generate-image.sh

```diff
@@ -66,9 +66,17 @@ cleanup_losetup() {
 generate_debian() {
     local -r image="${DEBIAN_IMAGE_ID}-x86-64.img"
 
-    mkosi --root-size=2G --distribution=debian --release="$DEBIAN_CODENAME" --format=gpt_ext4 --bootable --checksum \
-          --password password --package=openssh-server,dhcpcd5 --package grub-efi-amd64-signed \
-          --package shim-signed --package lsb-release --output="$image" build
+    mkosi --distribution=debian --release="$DEBIAN_CODENAME" --output="$image" \
```

Comment:
Q: why not have them all on each their own line now?

Reply:
Okey, I'll do.